files/oyo/Sprint1.2_A.ifc

```diff
@@ -0,0 +1,59 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition [Ifc4X4NotAssigned]'),'2;1');
+FILE_NAME(
+/* name */ '..\\..\\..\\Sprint1.2_A.ifc',
+/* time_stamp */ '2023-03-14T20:17:54',
+/* author */ ('Kazunori Takahashi'),
+/* organization */ ('OYO Corporation'),
+/* preprocessor_version */ 'GeometryGymIFC v0.2.14.0 by Geometry Gym Pty Ltd built 2023-03-14T10:36:14',
+/* originating_system */ 'Sprint1_2 v1.0.0.0',
+/* authorization */ 'Sprint1.2_A'
+);
+
+FILE_SCHEMA (('IFC4X4_17239AAA'));
+ENDSEC;
+
+DATA;
+#1= IFCDIRECTION((1.0,0.0,0.0));
+#2= IFCDIRECTION((0.0,1.0,0.0));
+#3= IFCDIRECTION((0.0,0.0,1.0));
+#4= IFCPROJECT('394HuS6Hz1YA_XfQJd0vhD',#10,'Test project',$,$,$,$,(#24),#11);
+#5= IFCPERSON($,'Takahashi','Kazunori',$,$,$,(#26),$);
+#6= IFCORGANIZATION($,'OYO Corporation',$,(#27),$);
+#7= IFCPERSONANDORGANIZATION(#5,#6,$);
+#8= IFCORGANIZATION($,'Geometry Gym Pty Ltd',$,$,$);
+#9= IFCAPPLICATION(#8,' v1.0.0.0','Sprint1_2 v1.0.0.0','Sprint1_2 v1.0.0.0');
+#10= IFCOWNERHISTORY(#7,#9,$,.ADDED.,1678792674,$,$,1678792674);
+#11= IFCUNITASSIGNMENT((#12,#13,#14,#17,#19,#20,#21));
+#12= IFCSIUNIT(*,.LENGTHUNIT.,$,.METRE.);
+#13= IFCSIUNIT(*,.AREAUNIT.,$,.SQUARE_METRE.);
+#14= IFCSIUNIT(*,.VOLUMEUNIT.,$,.CUBIC_METRE.);
+#15= IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#16= IFCMEASUREWITHUNIT(IFCPLANEANGLEMEASURE(0.0174532925199433),#15);
+#17= IFCCONVERSIONBASEDUNIT(#18,.PLANEANGLEUNIT.,'degree',#16);
+#18= IFCDIMENSIONALEXPONENTS(0,0,0,0,0,0,0);
+#19= IFCSIUNIT(*,.MASSUNIT.,.KILO.,.GRAM.);
+#20= IFCSIUNIT(*,.TIMEUNIT.,$,.SECOND.);
+#21= IFCMONETARYUNIT('EUR');
+#22= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#23= IFCAXIS2PLACEMENT3D(#22,#3,#1);
+#24= IFCGEOMETRICREPRESENTATIONCONTEXT('3D','Model',3,0.00000001,#23,#25);
+#25= IFCDIRECTION((0.0,1.0));
+#26= IFCACTORROLE(.CONSULTANT.,$,$);
+#27= IFCACTORROLE(.CONSULTANT.,$,$);
+#28= IFCPERSONANDORGANIZATION(#5,#6,$);
+#29= IFCACTOR('2w6q6ViCj5WxTN7WHsiMTo',#10,$,$,$,#28);
+#30= IFCRELDECLARES('0g5BZqHJj3KurdfQp0mOR3',#10,$,$,#4,(#29));
+#31= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#32= IFCAXIS2PLACEMENT3D(#31,#3,#1);
+#34= IFCSITE('0eXNiIZjn07PmpVCMfT14G',#10,'Test site',$,$,#35,$,$,$,$,$,$,$,$);
+#35= IFCLOCALPLACEMENT($,#23);
+#36= IFCRELAGGREGATES('1KaxC4LWdRO41qw833HO5p',#10,$,$,#4,(#34));
+#37= IFCCLASSIFICATION('classification source',$,$,'myClassification',$,$,$);
+#38= IFCRELASSOCIATESCLASSIFICATION('2Bbzqs65L6hPPH37xAhTzN',#10,$,$,(#4),#37);
+#39= IFCPROJECTEDCRS('EPSG:5834','DB_REF / 3-degree Gauss-Kruger zone 4 (E-N) + DHHN92 height','EPSG:5684','EPSG:5783','Gauss-Kruger','4',#12);
+#40= IFCMAPCONVERSION(#24,#39,4468005.0,5334600.0,515.0,1.0,0.0,1.0);

```

Comment:
I would have preferred you picked an example from Japan, but this is not wrong either :D

Reply:
We could totally do that:) Hold on...